projects/mysql-cluster/README.md

```diff
@@ -0,0 +1,222 @@
+<!--forhugo
++++
+title="MySQL Replication -  Configuration and Troubleshooting"
++++
+forhugo-->
+# MySQL Replication - Configuration and Troubleshooting
+
+This project is designed to introduce you to MySQL database setup, schema creation, replication, and failover using Amazon EC2 instances.
+
+You will set up a primary MySQL server, test it, add a secondary server for replication, and demonstrate replication and failover processes. Most steps are designed to succeed, but some will require some troubleshooting and problem-solving.
+
+## Learning Objectives
+- Install MySQL server
+- Configure MySQL as a cluster
+- Troubleshoot the cluster configuration
+- Fail-over the cluster
+
+Timebox: 5 days
```

Comment:
This feels quite a generous time for this project - I'd probably drop this to maybe 2 days?

Reply:
I was generous because I thought many things here would be very new experiences (such as dealing with EC2 instances for example). Btw are we pre-provisioning the instances for the students? I later found out that they don't have access to AWS

Reply:
Also I'll be honest I had to troubleshoot this myself and spent two days until it worked (not full days but still)

Reply:
Done

Reply:
They have their own AWS accounts, so we can either provision for them from ours and give them SSH, or we can just have them provision in their own.